Dockerfile

```diff
@@ -326,6 +330,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         less \
         libgomp1 \
         libsqlite3-0 \
+        ncbi-blast+ \
```

Comment:
Ah, just caught up on conversation in https://github.com/nextstrain/docker-base/issues/127. 

Worth noting this is installing https://packages.debian.org/bookworm/ncbi-blast+ which is v2.12.0.